website/about/index.html

```diff
@@ -0,0 +1,97 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title %}{{ _('About Us') }}{% endblock %}
+
+{% block page_desc %}
+{{ _('About the Thunderbird Project.') }}
+{% endblock %}
+
+{% block body_id %}contact{% endblock %}
+
+{% block page_css %}
+<link href="/media/css/thunderbird-features.css" rel="stylesheet" type="text/css"/>
+{% endblock %}
+
+{% block content %}
+<header id="main-feature">
+  <h1>{{ self.page_title() }}</h1>
+  <h2>{{ self.page_desc() }}</h2>
+</header>
+
+<nav role="directory">
+  <ul class="feature-links">
+    <li><a href="#what">{{ _('What is Thunderbird') }}</a></li>
+    <li><a href="#who"</a>{{ _('Who We Are')}}
+    <li><a href="#mozilla">{{ _('Thunderbird and Mozilla Foundation') }}</a></li>
+  </ul>
+</nav>
+
+<main role="main">
+  <section id="what">
+    <h2>{{ _('What is Thunderbird?') }}</h2>
+    <div class="row">
+      <div class="col full">
+        <p>{{ _('Software made to make email easier.') }}</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col full">
+        <section>
+            <p>{{ _('Thunderbird is a free and open source email, newsfeed, chat, and calendaring client, that\'s easy to set up and customize. One of the core principles of Thunderbird is the use and promotion of
```

Comment:
Use unicode single quotes, like in https://github.com/thundernest/thunderbird-website/blob/master/website/index.html#L9 - then you don't need to escape them either. Should be an easy search/replace for \'.

Reply:
Ah yes, it is changed.